scripts/haproxy.cfg

```diff
@@ -0,0 +1,37 @@
+# Any change in this file should be reflected in discovery-infra/start_discovery.py as well
+defaults
+  timeout client 30s
+  timeout server 30s
+  timeout connect 5s
+
+frontend https
+    bind :::443 v4v6
```

Comment:
Do we need IPv4 binding?

Reply:
will it work with cloud or staging dpeloyments?